apps/site/assets/ts/stop/__tests__/StopPageRedesignTest.tsx

```diff
@@ -161,10 +161,70 @@ describe("StopPageRedesign", () => {
     ).not.toBeNull();
   });
 
-  it("should only render current alerts and not future alerts", () => {
+  it("should only render closures, shuttles, and supensions", () => {
```

Comment:
This test doesn't test for suspensions.  One should be added

Reply:
This name needs to be changed, or an alert with the `suspension` status should be added to this test